templates/index.html

```diff
@@ -60,19 +60,19 @@
 
             <pre class="line"> </pre>
             <pre class="line"><span class="dyer-white">num_active_channels:</span></pre>
-            <pre class="line">{{num_active_channels}}</pre>
+            <pre class="line">{{num_active_channels || numActiveChannels}}</pre>
```

Comment:
I really have doubts about this. so is `num_active_channels` not there anymore?

Reply:
I've just had a setting wrong, now it's back